clang/include/clang/CConv/ConstraintVariables.h

```diff
@@ -404,10 +404,10 @@ class FunctionVariableConstraint : public ConstraintVariable {
   FunctionVariableConstraint(FunctionVariableConstraint *Ot,
                              Constraints &CS);
   // N constraints on the return value of the function.
-  ConstraintVariable *ReturnVar;
+  PVConstraint *ReturnVar;
```

Comment:
Nice. I recall thinking in the past that this was always true and we should refactor.